### DISCUSSION TITLE
Release 1.10.0 in july

### DISCUSSION THREAD

**Discussion by jeromerobert** [pdfarranger/pdfarranger#880] [Announcements]
Dear all,

If nobody objects, I think I'll release the next version in July. There are two features that I find quite exciting in this version (both from @kbengs): copy/paste as overlay/underlay and page merging. 

It's a good time to test and translate the main branch.

Notification of translators (let me know if you want to be removed from the list):
@albanobattistella, @Caesar, @emintufan, @ersen0, @hahnavi, @hugok79, @juanma1980, @juxuanu, @Knirps1, @kuroehanako, @Lrss, @manussos, @MichelePezza, @oscfdezdz, @ovari, @pavelzahradnik, @sicklylife-jp, @smarquespt, @sveinki, @Vistaus, @volkov0aa, @wcxu21, @yarons (not notified because of recent contributions: Ser82-png, milotype, Vovkiv, Mejans, mapi68)

Regards,

Jerome

**Comment by caesar** [comment]
I don't think I'm a translator as I wasn't aware of this project until now.
Looks like a nice project though! 😄

**Reply by jeromerobert** [reply]
Oups sorry, I fixed the list. It should have been "TulioCaesar".

**Reply by ghost** [reply]
Yay! Can't wait!

**Comment by juxuanu** [comment]
It would be very handy for us translators and for you developers if this was translated in a platform designed for it (poeditor, weblate, transifex...).

**Reply by jeromerobert** [reply]
I tried to use Transifex for an other project and I stopped using it because it was poorly interacting with git. I cannot find any free weblate hosting. I'll give poeditor a try as it's free for open source projects and seems to have a good Github integration. It won't probably be for that release.

**Reply by juxuanu** [reply]
I recommend setting up PR presets and calling their webhook with Actions, this way you will get PRs for new translations automatically.

**Reply by Vistaus** [reply]
As a translator, I like Transifex the best, but good ol' po file translating, like we are now doing, is great too. I'd say # 1 Transifex, # 2 old-school. :) But I could live with POEditor. It's not the best, but not the worst either.

**Reply by sveinki** [reply]
Don't see much benefit translation-wise in such a translation platform for a one-file project. Certainly it could attract more new translators, but then the quality of the translations may suffer.

Weblate has a hosting plan called 'Libre' specially for open source projects: https://hosted.weblate.org/hosting/
"Translated content has to be released under a license approved by OSI or recognized as libre by FSF."

**Reply by homocomputeris** [reply]
Crowdin would be the best choice, and it's free for FOSS projects.

**Reply by Vistaus** [reply]
Ugh, please no. Not only do I hae Crowdin's interface, it's also very problematic with notifications and stuff. And I've heard the same issues from other translators too. Crowdin's been a nightmare for years.

**Reply by homocomputeris** [reply]
At the very least Crowdin's interface isn't as slow as Transifex's, and search and glossary matching work much better.

**Reply by Vistaus** [reply]
Transifex can be a bit slow at times, true. But Crowdin is sometimes slow, too. Not so much in the translation interface, but in all that leads up to it.
Search doesn't work well in my experience (and I have a lot of experience with translating as it's also my job, and translation platforms, including Crowdin!), but I will agree that glossary matching is the one thing it has going for it. But I'm still voting against Crowdin.

**Comment by Vistaus** [comment]
Thanks for the notification! I'll update the Dutch translation today or tomorrow! 🙂

**Comment by ovari** [comment]
@jeromerobert
1. Could you please update the source so the Hungarian translation file will have something like `#fuzzy` or `blank`? This will enable forking your repository and adding translations using the web browser.
2. Could you please advise how many strings there are to translate so we know how many strings to search for?

Thank you

**Reply by ovari** [reply]
https://github.com/pdfarranger/pdfarranger/pull/898

**Comment by jeromerobert** [comment]
> Could you please update the source so the Hungarian translation file will have something like #fuzzy or blank? This will enable forking your repository and adding translations using the web browser.

Done already (b1c4ef654cabe2e3d37bf25029a34d9329342dcd and e966caa523134d2bd8ca734b8593c76f41661b6a). Since then, only the line numbers in the source code have changed.

> Could you please advise how many strings there are to translate so we know how many strings to search for?
```
$ grep 'msgstr ""' po/hu.po | wc -l
39
```

**Reply by ovari** [reply]
Can you please merge https://github.com/pdfarranger/pdfarranger/pull/891 so we can delete the fork and then fork again to do translations? Thank you

**Reply by jeromerobert** [reply]
No I can't because I don't understand what you are trying to do.

**Comment by jeromerobert** [comment]
Done
